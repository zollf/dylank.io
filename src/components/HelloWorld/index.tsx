import React from 'react';
import HomeIcon from '@/images/home.svg';

import styles from './styles.module.scss';

const HelloWorld = () => {
  return (
    <div className={styles.helloWorld}>
      <HomeIcon />
      Hello World
    </div>
  );
};

export default HelloWorld;
