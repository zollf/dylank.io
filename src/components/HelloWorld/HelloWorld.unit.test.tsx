import React from 'react';
import { render } from '@testing-library/react';

import HelloWorld from './';

it('matches its snapshot', () => {
  expect(render(<HelloWorld />).asFragment).toMatchSnapshot();
});
