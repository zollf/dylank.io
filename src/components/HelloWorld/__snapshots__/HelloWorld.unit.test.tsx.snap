// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches its snapshot 1`] = `[Function]`;
